hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -846,10 +846,17 @@
     </description>
   </property>
   <property>
-    <name>ozone.scm.datanode.max.pipeline.engagement</name>
-    <value>5</value>
+  <name>ozone.scm.datanode.max.pipeline.engagement</name>
+  <value>0</value>
+  <tag>OZONE, SCM, PIPELINE</tag>
+  <description>Max number of pipelines per datanode can be engaged in.
```

Comment:
Can we add some description for the meaning of default 0?

Reply:
Sure.